index.html

```diff
@@ -1,96 +1,227 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-	<meta charset="utf-8">
+	<meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
Do you know what the function of this is? Does it surprise you to know your code would work without it? Even though it is good to have.